android/src/main/java/com/clevertap/clevertap_plugin/CleverTapPlugin.java

```diff
@@ -1365,7 +1366,12 @@ private void showInbox(MethodCall call, Result result) {
         }
     }
 
-    private void resetUser() {
-        cleverTapAPI.resetUser();
+    private void resetUser(Result result) {
+        cleverTapAPI.resetUser(new CTEventNotifier(){
+            @Override
+            public void onEventComplete(){
+                result.success(null);
+            }
+        });
```

Comment:
Why don't we give a callback for `onEventFailed()`? We can add another event to log this. Check this comment here: https://code.nurture.farm/afs-spraying/afs-farmer/-/merge_requests/814#note_46031

Reply:
have not added as clevertap itself was not handling any error specifically.So could not think of a scenario..but kept a method if if we find a use case in future..We can add a method with a flat error message. What do you think?